roles/change-paths/tasks/main.yml

```diff
@@ -193,3 +193,10 @@
 - name: Save modified JS Injector rules to disk
   shell: "{{ drush_alias }} scr {{ scripts_dir }}/js-injector.php"
   notify: Clear site cache
+
+# Bring 'er out of hyperspace, Chewy
+- name: Turn off maintenance mode for Sites 1.x site and ACSF site
+  shell: "{{ item }} -y --exact vset maintenance_mode 0"
+  with_items:
+    - " {{ drush_alias }}"
+    - " drush @{{ server_alias }}.{{ site_prefix }}_{{ inventory_hostname }} "
```

Comment:
I can adjust my aliases much easier than trying to adjust this code. I'll change it, but it is a good lesson learned for future scripts is that we cannot assume each other's aliases.

Reply:
Yeah there are a few things at play here, lessons learned over a year of iterating on this software package.

1. We should have uniquely namespaced the drush aliases (for both on-prem and ACSF) that these Ansible playbooks use.
2. I have inadvertently been pushing a transition away from OS X and one's everyday development environments, to `my-stanford-vbox` and `sites-uat`, which are Ubuntu/Debian, and more purposefully tuned for this work.

Sorry for the friction.

Reply:
Oh. Yeah.

https://github.com/SU-SWS/ansible-playbooks/blob/master/roles/download-site/tasks/main.yml#L39

I can add that in here.